modules/compliance-removing-kubeletconfig.adoc

```diff
@@ -58,6 +58,11 @@ status:
 <1> The scan name of the remediation.
 <2> The remediation that was added to the `KubeletConfig` objects.
 +  
+[NOTE]
+====
+Due to a known issue, if the remediation invokes an `evictionHard` kubelet configuration, you must specify all of the `evictionHard` parameters: `memory.available`, `nodefs.available`, `nodefs.inodesFree`, `imagefs.available`, and `imagefs.inodesFree`. If you do not specify all parameters, only the specified parameters are applied and the remediation will not function properly.
```

Comment:
I guess we should not mention due to a known issue as based on Ryan's comment in bug, all the required fields should be specified if setting the evictionHard option.